initial code and readme for blog post

codeforger-wp opened the pull request:
For brevity, I am unconcerned with Sanitization of inputs, or escaping of outputs. Unless one of you is particularly keen on it.

Commit:
initial code and readme for blog post

iamthomasbarrett commented on index.php:
```diff
@@ -0,0 +1,63 @@
+
+sleep( 1 );
+?>
+
```
Best not to have whitespace before doctype.

iamthomasbarrett commented on index.php:
```diff
@@ -0,0 +1,63 @@
+		<h1> Requester Info </h1>
+		<p>This page demonstrates the calling conventions for object singletons and static classes.</p>
+		<p>We expect that both columns have the same values, as we are demonstrating that these two techniques for global data containers are equivalent.</p>
+		<table>
```
Did you want to make full use of thead and tbody too?

iamthomasbarrett commented on object-singleton.php:
```diff
@@ -0,0 +1,60 @@
+<?php
+
```
Do you want it to be commented?

codeforger-wp replied:
As the code is explained in full in the blog post, do you think its also worth having comments in the code? I guess if people land directly on this code base, then comments might be usefull. Ill add some

Commit:
Review Updates:

Remove whitespace before <!doctype>

Make use of <thead> and <tbody> for the table

Commit:
Review Update

Add light comments to explain the main anatomy of our two classes
